sources/views/table.js

```diff
@@ -0,0 +1,39 @@
+import {JetView} from "webix-jet";
+
+export default class Table extends JetView {
+	constructor(app, name, data) {
+		super(app, name);
+		this._gridData = data;
+	}
+
+	config() {
+		return {
+			rows: [
+				{
+					view: "datatable",
+					autoConfig: true,
+					localId: "table"
+				},
+				{
+					cols: [
+						{view: "button", value: "Add", click: () => this.addItem(this.$$("table"))},
+						{view: "button", value: "Delete", click: () => this.deletedItem(this.$$("table"))}
+					]
+				}
+			]
+		};
+	}
+
+	init(view) {
+		view.queryView("datatable").parse(this._gridData);
```

Comment:
you can access datatable by `this.$$("table")` in any method of this class, add/remove item as well - you don't need to pass its instance from click handler

Reply:
in init as well :)